controller/Makefile

```diff
@@ -107,6 +107,7 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 .PHONY: manifests
 manifests: controller-gen ## generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	@$(MAKE) -s chart-crds
```

Comment:
This should ensure that when we regenerate base crds, controller chart copies are being updated as well.